client/js/entities/model/bioticState.js

```diff
@@ -28,9 +28,7 @@ return function BioticState(entityState, health) {
 
 	self.prettyCloseTo = function (other) {
 		return closeVec(self.entityState.body.pos, other.entityState.body.pos)
-			&& closeVec(self.entityState.body.dir, other.entityState.body.dir)
-			&& closeVec(self.entityState.body.vel, other.entityState.body.vel)
-			&& close(self.health.life, other.health.life);
+			&& closeVec(self.entityState.body.vel, other.entityState.body.vel);
```

Comment:
They never will. Ever. Either the direction isn't changing (and thus, they match) or the client is changing the direction, and then it won't ever match.

We would want this in a case where the server forces the client to look a particular direction... but we don't have anything like that right now.